dev_setup.sh

```diff
@@ -369,6 +369,15 @@ function fedora_install() {
     $SUDO dnf install -y git python3 python3-devel python3-pip python3-setuptools python3-virtualenv pygobject3-devel libtool libffi-devel openssl-devel autoconf bison swig glib2-devel portaudio-devel mpg123 mpg123-plugins-pulseaudio screen curl pkgconfig libicu-devel automake libjpeg-turbo-devel fann-devel gcc-c++ redhat-rpm-config jq make
 }
 
+function blackpanther_install() {
+    if [ -x /usr/bin/sudo ];then
+	$SUDO installing git python3 python3-devel python3-pip python3-setuptools python3-virtualenv python-gobject3-devel libtool lib64ffi-devel openssl-devel autoconf bison swig glib2-devel portaudio-devel mpg123 mpg123-pulse screen curl pkgconfig lib64icu-devel automake lib64jpeg-devel lib64fann-devel gcc-c++ jq make
+    else
+	# own authentication without sudo
+	installing git python3 python3-devel python3-pip python3-setuptools python3-virtualenv python-gobject3-devel libtool lib64ffi-devel openssl-devel autoconf bison swig glib2-devel portaudio-devel mpg123 mpg123-pulse screen curl pkgconfig lib64icu-devel automake lib64jpeg--devel lib64fann-devel gcc-c++ jq make
+    fi
```

Comment:
I'm not sure that the else path on this conditional will ever be executed.

If sudo isn't on the system then the `found_exe sudo` up around line 129 will fail and the script will exit. Have you tested this path out? I'm not at all familiar with this distro or the packaging manager it uses.

We do handle this better in [Pako](https://github.com/mycroftAI/pako) now, which is what we use for installing system packages for Skills etc.

Reply:
It's ok, because the blackPanther don't use sudo for authentication on the default installed system, but contain a wrapper on different path (the name is sudo), and system allows install sudo package if user want to this path: /usr/bin/sudo. Another way for 'detect', example  if found_exe find it the autentication will not work on blackPanther OS.  (or requires sudo install too)
Sorry I I'm too late for response, we work on many project in same time in our distro.